src/views/Calendar.vue

```diff
@@ -254,6 +256,22 @@ export default {
 				})
 			}
 		}, 1000 * 60)
+
+		this.updateTodayJob = setInterval(() => {
+			const newDate = getYYYYMMDDFromFirstdayParam('now')
+
+			if (this.updateTodayJobPreviousDate === null) {
+				this.updateTodayJobPreviousDate = newDate
+				return
+			}
+
+			if (this.updateTodayJobPreviousDate !== newDate) {
+				this.updateTodayJobPreviousDate = newDate
+
+				const calendarApi = this.$refs.fullCalendar.getApi()
+				calendarApi.render()
+			}
+		}, 1000)
```

Comment:
No idea how much computation this needs, but wouldn't it be enough to do it every minute or so? I guess checking every second is not really necessary, since 86399 times out of 86400 nothing has to be done.

Reply:
I clicked around and couldn't tell any performance difference.

I would personally prefer the more accurate version of updating it immediately, because that's what people are used to from other calendaring software.

Reply:
I also tried to reduce write access by only updating `this.updateTodayJobPreviousDate` when it actually changed, not every second. So we don't unnecessarily cause any Vue watchers to run.

Reply:
Fine with me.